README.md

```diff
@@ -1 +1,8 @@
-# DesignPatterns
\ No newline at end of file
+
+List of patterns:
+
+1)Factory Pattern
```

Comment:
1. Why is mobile and interface? HOw does inheritance work here?
2. Why is paymentApp a function of mobile?
3. Would one factory be enough for all the subtypes of phones?